content/blog/2022-05-31-may-22-community-gems.md

```diff
@@ -0,0 +1,188 @@
+
+Very interesting question from @mathematiguy!
+
+One way to do this is to have all `foreach` stages write out to different paths
+within the same directory and then track the entire directory as a dependency of
+your downstream stage.
+
+Here's an example of how that might look in your `dvc.yaml` file.
+
+```yaml
+stages:
+  cleanups:
+    foreach:
+      - raw1
+      - labels1
+      - raw2
+    do:
+      cmd: echo "${item}" > "data/${item}"
+      outs:
+        - data/${item}
+
+  reduce:
+    cmd: echo file > file
+    deps:
+      - data
+    outs:
+      - file
+```
+
+### [Is there a way to version and move data from one cloud storage to another with DVC remotes?](https://discord.com/channels/485586884165107732/563406153334128681/968778284114538496)
+
+Wonderful question from @Hisham!
+
+There are a couple of ways you can do this. One approach is to use
+`dvc add --to-remote`.
+
+The other approach is to use the
+[`import-url`](https://dvc.org/doc/command-reference/import-url#example-transfer-to-remote-storage)
+functionality. You can see an example of how to do this in the docs. Just make
+sure that you have your remotes set up!
+
+### [If I'm using Feast feature store, is it possible to version datasets with DVC?](https://discord.com/channels/485586884165107732/563406153334128681/968899175561449532)
+
+This is a good integration question from @Bernardo Galvao!
+
+If you want to fetch historical features from the offline store to generate
+training data, a typical pattern would be to write the script to do so and set
+up a DVC pipeline stage to track that script and version the output file. This
+is similar to how a lot of people use DVC alongside SQL databases.
+
+### [How can I run a DVC pipeline in a Docker container?](https://discord.com/channels/485586884165107732/563406153334128681/969640280263389184)
+
+Nice question from @Anudeep!
+
+Here's an example of a Dockerfile with a simple DVC setup.
+
+```docker
+FROM ubuntu:latest
+RUN apt-get update && apt install -y python-is-python3 python3-pip
+WORKDIR /dvc_project
+
+COPY . .
+pip install -r requirements.txt # assuming your requirements, including dvc, are here
+CMD dvc pull && dvc exp run
+```
+
+You would save this file and then run the following commands in your terminal.
+
+```dvc
+$ docker build -t "myproject-dvc-exp-run" .
+$ docker run myproject-dvc-exp-run
+```
+
+You could also use the `dvc repro` command or any of the other DVC commands.
+
+### [How can I reset a repository and start fresh with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/970344379938127892)
+
+Nice question from @strickvl!
+
+There are a couple of ways to do this.
+
+You can remove all the `.dvc` files and the `.dvc` folder in the root directory
+manually, but keep in mind that **any data versioned with DVC will be deleted**,
+since it's stored in `.dvc/cache`. After that you can re-initialize the repo
+with `dvc init`.
```

Comment:
@dtrifiro Did you have a use case in mind where you think this is preferable to `dvc destroy`, or were you just unfamiliar with that command? I would probably just recommend `dvc destroy` unless we have a scenario in mind where the manual approach would be better.

Reply:
I think it's just better to use `dvc destroy`, I just forgot it existed

Reply:
@flippedcoder I would recommend removing this paragraph then.